ui/components/resolvers/SubscriptionsDropDown.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Query } from "react-apollo";
+
+import {
+    MenuItem,
+    InputGroup,
+    FormControl,
+    DropdownButton,
+    Col
+} from "patternfly-react";
+
+import GetSubscriptions from "../../graphql/GetSubscriptions.graphql";
+
+import { controlLabel, formControl } from "./SubscriptionsDropDown.css";
+
+const SubscriptionsDropDown = ({ selected, onSubscriptionSelect }) => (
+    <React.Fragment>
+        <Col sm={2} className={controlLabel}>Subscription</Col>
+        <Col sm={6}>
+            <Query query={GetSubscriptions} variables={undefined}>
+                {({ loading, error, data }) => {
+                    if (!data || !data.subscriptions || !data.subscriptions.length) {
+                        return <FormControl.Static>Please first create a subscription</FormControl.Static>;
+                    }
+
+                    const options = [
+                        <MenuItem key="empty_subscription" eventKey={undefined} />,
```

Comment:
It is hard to select the empty option from dropdown. Maybe add a text to it (e.g. "None")?